dev/com.ibm.ws.org.apache.cxf.cxf.api.2.6/bnd.bnd

```diff
@@ -14,7 +14,6 @@ cxfVersion=2.6.2
 
 -includeresource: \
    @${repo;org.apache.cxf:cxf-api;${cxfVersion}}!/!*-INF/*,\
-   com/ibm/ws=${bin}/com/ibm/ws,\
```

Comment:
I removed the source for the NoOpActivator because jaxws-2.2 has never had a need to over-write the OSGi functionality that ships with CXF 2.6.2.

Reply:
transports.http already has an Activator as well